### PR TITLE
Feature 124/prepaid expenses

### LINKED ISSUE #124: Add Prepaid Expense column
Add Prepaid expense column to the Wages page and account for it in the calculations

### PR DESCRIPTION
**Pull request by alisondraV**
## Description

Add pre-paid expenses column to the Wages page.

## Issue

Fixes #124

### CODE REVIEW

File: .eslintrc.json [hunk 1]
```diff
@@ -4,7 +4,7 @@
     "quotes": ["error", "single", { "avoidEscape": true }],
     "semi": ["error", "never"],
     "object-curly-spacing": ["error", "always"],
-    "no-unused-vars": ["warn"],
+    "no-unused-vars": 0,
```
Comment (ANDREYDEN) [inline .eslintrc.json:7]: Why is this off now?

File: pages/employees/salary.tsx [hunk 2]
```diff
@@ -66,10 +81,14 @@ const Salary: NextPage = () => {
         salesIncomeTotalsLoading,
         salesIncomeTotalsError,
     } = usePosterGetSalesIncomeForEmployees(employees, shifts)
-    
+
+    // TODO: refactor repetition
```
Comment (ANDREYDEN) [inline pages/employees/salary.tsx:85]: yes

**Review by ANDREYDEN — CHANGES_REQUESTED** [review]

File: .eslintrc.json (hunk 1 above)
Comment (alisondraV) [reply .eslintrc.json:7]: Cause it was causing a bunch of linting warnings from the interfaces

Comment (ANDREYDEN) [reply .eslintrc.json:7]: can you please either make it so that the interfaces don't cause warnings and the rule stays or create a ticket for this?

Comment (alisondraV) [reply .eslintrc.json:7]: Sure, sounds good

**Review by ANDREYDEN — APPROVED** [review]

**Pull request merged**